SKILL.tsv

```diff
@@ -1673,7 +1673,7 @@ SKILL_20150317_001672	Spreads poison to nearby enemies. Running and item usage i
 SKILL_20150317_001673	Growth
 SKILL_20150317_001674	Confusion
 SKILL_20150317_001675	Can't distinguish friend and foe.
-SKILL_20150317_001676	Stiffen
+SKILL_20150317_001676	Staggered
```

Comment:
I've explained it in Discord, but saying it here for @zhouyu47. In this context, it can also mean stagger.